WcaOnRails/app/views/incidents/index.html.erb

```diff
@@ -0,0 +1,108 @@
+<% provide(:title, "Incidents log") %>
+
+<%= render layout: "nav" do %>
+  <h1><%= yield(:title) %></h1>
+
+  <div id="custom-search-toolbar">
+    <input id="incident-tags" class="form-control" type="text" placeholder="Filter by tags">
+  </div>
+  <div class="incidents-log-table-container">
+    <%= wca_table(table_class: "incidents-log-table", data: { toggle: "table", pagination: true, search: true, toolbar: "#custom-search-toolbar" }, floatThead: false, greedy: false) do %>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th class="tags-cell">Tags</th>
+          <th class="comps-cell">Happened during</th>
+          <th class="status-cell">Status</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @incidents.each do |incident| %>
+          <tr>
+            <td><%= link_to incident.name, incident %></td>
+            <td data-tags="<%= incident.tags %>">
+              <% incident.tags_array.each do |t| %>
+                <%= render "incident_tag", tag: t %>
+              <% end %>
+            </td>
+            <td data-tags="<%= incident.competitions %>">
```

Comment:
What is this used for? Also `data-tags` seems wrong in the context of `incident.competitions`.

Reply:
Not used yet, though I plan to add a per-competition filter, similar to the tag filter.
I'll rename the data attribute though ^^